contracts/OffChainDebts.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.4.15;
+
+contract OffChainDebts {
+ 
+    event BorrowLogger(address addr, uint ammount, uint balance);
+    event RepayLogger(address addr, uint ammount, uint balance);
+    mapping (address => uint) public debts;
+    address owner;
+    
+    function OffChainDebts() public {
+        owner = msg.sender;
+    }
+
+   // Called by borrower. Borrower can borrow multiple times.
+   // @param ammount - money borrowed (in USD)
+   function borrow(uint ammount) public {
+        uint newAmmount = debts[msg.sender] + ammount;
+        require(newAmmount >= debts[msg.sender]);
+        debts[msg.sender] = newAmmount;
+        BorrowLogger(msg.sender, ammount, debts[msg.sender]);
+   }
```

Comment:
It would be easier to test or debug the function if it explicitly returns something.